Lesson09/scripts/hw19.js

```diff
@@ -0,0 +1,71 @@
+console.log ("Hello, World!");
+
+//ДЗ 19. Створити та описати сутності Багатоквартирного будинку, квартири, мешканця.
+
+//Додати можливість створювати нові будинки на певну кількість квартир.
+
+//Не обмежувати кількість мешканців у квартирі
+
+class Person {
+    namePerson;
+    genderPerson;
```

Comment:
```suggestion
    name;
    gender;
```

почитайте - я постив про неймінг
ви в цих назвах дублюєте контекст. Ці поля вже належуть класу Людина